.agents/skills/context7/SKILL.md

```
@@ -0,0 +1,135 @@
+  [Source](https://github.com/...)
+  ```code example```
+
+### Usage Patterns
+- Pattern description — when to use, gotchas.
+  [Source](https://github.com/...)
+  ```code example```
+
+### Configuration
+- Config option — description.
+  [Source](https://github.com/...)
+  ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)